### DISCUSSION TITLE
Fortinet firewall SNMP v3 configuration

### DISCUSSION THREAD

**Discussion by ghost** [prometheus/snmp_exporter#1285] [Q&A]
Does anyone have an idea why my snmpwalk command is getting time out for V3 ?

**Comment by bastischubert** [comment]
hi @santosh4tec could be anything - can you add some more context what exactly you're executing and whats the result. also please not that snmpwalk is not maintained by the snmp_exporter devs 8-)

**Reply by ghost** [reply]
snmp exporter unable to scrape metrics @bastischubert  from network device

**Reply by bastischubert** [reply]
do you have some more detailed output from snmp_exporter (run it with log level debug and snmp.debug-packets)

**Reply by HassanNetops** [reply]
@santosh4tec If its timing out the issue is on the fortigate end. Add the VM IP to trusted host in fortigate and check routing.
